From c323ea8204b7ffebd114610fcc1886c85ae6f8b8 Mon Sep 17 00:00:00 2001
From: Rashed Abdel-Tawab <rashed@linux.com>
Date: Mon, 29 Oct 2018 00:01:47 -0700
Subject: [PATCH 21/21] config: Move vendor/extra inclusion before Lineage
 configuration

This allows overriding Lineage configs with custom configs in vendor/extra
(ex. a custom default alarm and notification sound)

Change-Id: I0f796b1184035e6c348799cc28a5864f7f5055b7
---
 config/common.mk | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/config/common.mk b/config/common.mk
index 6efa9125..770a91c4 100644
--- a/config/common.mk
+++ b/config/common.mk
@@ -1,3 +1,5 @@
+$(call inherit-product-if-exists, vendor/extra/product.mk)
+
 PRODUCT_BRAND ?= LineageOS
 
 PRODUCT_BUILD_PROP_OVERRIDES += BUILD_UTC_DATE=0
@@ -367,5 +369,3 @@ endif
 
 -include $(WORKSPACE)/build_env/image-auto-bits.mk
 -include vendor/lineage/config/partner_gms.mk
-
-$(call inherit-product-if-exists, vendor/extra/product.mk)
-- 
2.17.1

